File: uri/bonus/bonus.js
var input = require('fs').readFileSync('/dev/stdin','utf8');
var lines = input.split('\n');

function bonus( salario, montante)
{
    return (salario + ((montante * 15)/100));
}

var nome = lines.shift();
var salario = parseFloat( lines.shift());
var montante = parseFloat( lines.shift());
console.log('TOTAL = R$ '+ bonus(salario, montante).toFixed(2) );

File: uri/bonus/bonus.rb
def calculabonus( salario, montante)
    return salario + ((montante * 15) / 100)
end

nome = gets.chomp
salario = gets.to_f
montante = gets.to_f
res = calculabonus(salario, montante)
puts "TOTAL = R$ %.2f" % [res]
